log/logger.go

```diff
@@ -0,0 +1,68 @@
+package log
+
+import (
+	"github.com/op/go-logging"
+	"io"
+)
+
+// BaseLogger is a convenience interface that makes use of all functionality from go-logging's
+// Logger struct. In addition, it defines Disable() and SetLevel(level) functions
+type BaseLogger interface {
```

Comment:
May I suggest to name it simply `Logger`?

Reply:
Sure, makes sense :slightly_smiling_face: . Done.